src/components/CustomerQuota/CheckoutSuccess/CheckoutSuccessCard.tsx

```diff
@@ -181,16 +191,13 @@ export const CheckoutSuccessCard: FunctionComponent<CheckoutSuccessCard> = ({
             <AppText style={sharedStyles.statusTitleWrapper}>
               <AppText style={sharedStyles.statusTitle}>{title}</AppText>
               {showGlobalQuota &&
-                quotaResponse!.globalQuota!.map(
-                  ({ quantity, quotaRefreshTime }, index: number) =>
-                    quotaRefreshTime ? (
-                      <UsageQuotaTitle
-                        key={index}
-                        quantity={quantity}
-                        quotaRefreshTime={quotaRefreshTime}
-                      />
-                    ) : undefined
-                )}
+              globalQuota!.quantity &&
```

Comment:
Does this mean that if the quantity is 0, we don't want to show the title as well?

Reply:
Good point, the message should still be shown if the quantity is 0. Will remove this!